automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -107,7 +169,7 @@ class MethodConfigSpec extends FreeSpec with Matchers with WebBrowserSpec with W
   private def generateInputsJson(inputs: Map[String, String]): File = {
     val file = File.createTempFile("MethodConfigSpec_", "_inputs.json")
     val writer = new PrintWriter(file)
-    val rows = inputs map { case (k, v) => s""""$k": "$v"""" }
+    val rows = inputs map { case (k, v) => s""""$k": $v""" }
```

Comment:
This seems odd. Did it not work before?

Reply:
No all values should have quotes (e.g. numbers) so quotes are now included in the values of the `variables` map when appropriate.